Scoop.js

```diff
@@ -1367,7 +1367,7 @@ export class Scoop {
   /**
    * @typedef {Object} ScoopCaptureSummary
    * @property {int} state
-   * @property {object} states - Possible values of Scoop.state
+   * @property {string[]} states - Indexed Scoop.states values.
```

Comment:
Is it worth saying "Zero-indexed," or is that obvious>

Reply:
Fixed :) !